### PR TITLE
missing file...changed it to a different file

### PR DESCRIPTION
**Pull request by chlocarlos**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ConvertJsonToCsv.java [hunk 1]
```diff
@@ -82,25 +84,18 @@ public class ConvertJsonToCsv {
...
     	    .addField("exchange", "//array/object/e")
     	    .addField("price", "//array/object/l")
     	    .addField("change", "//array/object/c")
     	    .addRecordBreak("//array/object");
-        DataWriter writer = new CSVWriter(output);
+
+		OutputStreamWriter output = new OutputStreamWriter(System.out);
+
+		DataWriter writer = new CSVWriter(output);
+
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/ConvertJsonToCsv.java:96]: Let's collapse these down into a single line.

File: .gitignore [hunk 2]
```diff
@@ -20,3 +20,4 @@ temp/
 WriteAParquetFile.parquet
 .WriteAParquetFile.parquet.crc
 src/main/resources/NorthConcepts-DataPipeline.license.old
+src/main/resources/NorthConcepts-DataPipeline.license
```
Comment (DeleTaylor) [inline .gitignore]: Hmmm...I thought I saw this change already. Do you need to pull master into your branch first?

BTW, it looks like you might have forked this repo and then are sending PRs from your master to the repo's master. This is okay, but we take different approach here. In the new approach you will clone this repo, create a branch (based on our naming pattern), commit your changes to the [feature] branch, and create a PR to merge your feature branch to master.

Here's a more detailed explanation:
https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (chlocarlos) [reply .gitignore]: This was a different example from the first one. The file path was incorrect.
yes, I forked the repo...ill start using the new approach

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ConvertJsonToCsv.java (hunk 1 above)
Comment (chlocarlos) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/ConvertJsonToCsv.java:96]: working on it

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**